src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,10 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class IntakeConstants {
+        public static double kIntakePower = something;
+    }
+    public static final int VictorSPXPort = 2;
```

Comment:
I would recommend including this in the IntakeConstants

Reply:
Should I also include the VictorSPXPort in IntakeConstants?

Reply:
Yes

Reply:
Ok, I made this change as well as other suggested changes, may you please take a look when you get a chance?

Reply:
Looks good

Reply:
Alright, thank you!